index.html

```diff
@@ -2733,6 +2733,29 @@ <h2 id="OfflineAudioCompletionEvent">
           </dd>
         </dl>
         <section>
+          <h2>Dictionaries</h2>
+          <dl title="dictionary AudioNodeChannelOptions" class="idl">
```

Comment:
Maybe the dictionary name should just be `AudioNodeOptions` instead of `AudioNodeChannelOptions`?